code/modules/mentorhelp/mentor.dm

```diff
@@ -74,34 +74,29 @@ GLOBAL_PROTECT(mentor_href_token)
 	while(mentors_regex.Find(mentors_text))
 		new /datum/mentors(mentors_regex.group[1])
 	if(!CONFIG_GET(flag/mentor_legacy_system) || dbfail)
-		var/datum/db_query/query_load_mentors = SSdbcore.NewQuery("SELECT `ckey`, FROM [format_table_name("mentor")]")
+		var/datum/db_query/query_load_mentors = SSdbcore.NewQuery("SELECT ckey FROM [format_table_name("mentor")] ORDER BY ckey")
```

Comment:
Fixing an SQL error, lmao. I still think it's a good idea to be explicit when specifying column names, i.e. <code>SELECT \`ckey\` FROM [format_table_name("mentor")] ORDER BY \`ckey\`</code>. Also probably specify the order in which to sort (`DESC`/`ASC`)?